app/posts/apply-for-teacher-training/2022-11-09-changing-how-we-ask-about-disabilities-and-health-conditions.md

```diff
@@ -0,0 +1,75 @@
+date: 2022-11-09
+ogImage:
+  src: /apply-for-teacher-training/changing-how-we-ask-about-disabilities-and-health-conditions/disability-question-changes.png
+  alt: "Illustration showing that previously there was an ‘Are you disabled?’ question which led to a ‘What disabilities do you have?’ question, and now there is a single ‘Do you have any of these disabilities or health conditions?’ question"
+related:
+  items:
+    - text: Initial design for equality monitoring in 2019
+      href: /apply-for-teacher-training/equality-monitoring/
+---
+
+![Illustration showing that previously there was an ‘Are you disabled?’ question which led to a ’What disabilities do you have?’ question, and now there is a single ‘Do you have any of these disabilities or health conditions?’ question](disability-question-changes.png)
+
+We’ve changed the way we ask candidates about disabilities and health conditions in the equality and diversity monitoring questions shown just before submitting an application.
+
+These changes have resulted in 3 times more people selecting a disability or health condition.
+
+## The issue
+
+Previously, we asked the question ‘Are you disabled?’ with the options ‘Yes’, ‘No’ and ‘Prefer not to say’.
+
+![Screenshot showing page asking ’Are you disabled?’ with the options ‘Yes’, ‘No’ and ‘Prefer not to say’](are-you-disabled.png "Previous design for first question")
+
+If a candidate selected Yes, then we presented a list of checkboxes, with the question ‘What disabilities do you have?’
+
+![Screenshot showing page with the question ‘What disabilities do you have?’ and this list of options: Blind or a serious visual impairment not corrected by glasses, Dear or a serious hearing impairment, Learning difficulty for example dyslexia, Long-standing illness for example cancer, Mental health condition for example depression, Physical disability or mobility issue for example impaired use of arms or legs, Social or communication impairment for example Asperger’s and Other.](disabilities-before.png "Previous design for second question")
+
+The advantage of asking the question ‘Are you disabled?’ on the first page is that it allowed the candidates to avoid having to read the list of checkboxes on the second page.
+
+However this caused an issue because candidates who would have chosen some of the checkboxes may not consider themselves as disabled. This meant they would select ‘No’ on the first page which would then skip the second page. We had some evidence of this from user research sessions, where one user who was dyslexic commented that not everyone considers this a disability.
+
+In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority [updated their recommended list of options](https://www.hesa.ac.uk/collection/c22053/revision_history), with some wording changes and 1 new option.
```

Comment:
It's not clear why you're telling the reader this. Presumably because it means you will need to update with the new options?

Reply:
Changed to:

> In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority [updated their recommended list of options](https://www.hesa.ac.uk/collection/c22053/revision_history), with some wording changes we needed to consider and 1 new option we needed to add.